<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>161</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>fghjfghk678768</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>253</x>
          <y>50</y>
          <width>111</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string>hjp9p90b</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>303</x>
          <y>80</y>
          <width>141</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>455o[6890-84856u</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>150</y>
          <width>121</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>adfadsfaf</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="toolButton_up">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>173</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>up</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_down">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>210</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>down</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
